### PR TITLE
Get box user instructions: first step

### PR DESCRIPTION
**Pull request by HalaKabalan1**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [x] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description

This is the first step in transferring instruction manuals to GitHub. It is not final and not ready to merge as there are still issues with the layout. Submitting it for reviewing and testing!

### CODE REVIEW

**Comment by lePereT** [comment]
Thank you for this substantial first pull request! I'm accepting as is and we can refine in future focused pull requests.

**Review by lePereT — APPROVED** [review]

File: docs/get-box/1-Contents.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+date: 2024-02-12
+category: document
+author: Rich Thanki
+order: 1
+---
+<style>h1,h2,h3,h4 { border-bottom: 0; } </style>
+
+# CONTENTS
+
+Welcome to Get Box, an easy-to-use Wi-Fi connectivity device. This guide explains how to set up your Get Box. Please read carefully before starting.
```
Comment (Ljangala) [inline docs/get-box/1-Contents.md:11]: Change to "internet connectivity device" please

File: docs/get-box/1-Contents.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+![GB_Full](GB_SVGs/GB_Complete_Box.svg)
+
+### This Manual Includes: 
+1. Getting Started: Setting Up and Using Get Box
+2. Get Box Light Guide
+3. Troubleshooting Get Box
+4. Safety Information and Certification
```
Comment (Ljangala) [inline docs/get-box/1-Contents.md:19]: Make this section friendlier please

**Review by Ljangala — APPROVED** [review]

**Comment by lePereT** [comment]
Seems it's all good to merge @HalaKabalan1

**Pull request merged**